samples/Sandbox/project.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true,
+    "preserveCompilationContext": true
+  },
+
+  "dependencies": {
+    "Microsoft.AspNetCore.HttpOverrides": "1.0.0-*",
+    "Microsoft.AspNetCore.IISPlatformHandler": "1.0.0-*",
+    "Microsoft.AspNetCore.Mvc": "1.0.0-*",
```

Comment:
:cry: The value here is having the sandbox in this repo for ease of testing. I could technically use the existing MVC apps in Entropy to verify.

Hmm, would this actually be a problem?. If an MVC breaking change occurs it'll break `RazorTooling`s build but not its existing package (since it's only dependent on MVC from a samples perspective). MVC doesn't have any src dependencies directly on `dotnet-razor-tooling` and even from a samples perspective MVC never directly executes or compiles `dotnet-razor-tooling`. This shouldn't be any different than `Entropy` unless I'm missing something.